desktop-widgets/tab-widgets/maintab.cpp

```diff
@@ -339,6 +344,9 @@ void MainTab::enableEdition(EditMode newEditMode)
 	MainWindow::instance()->dive_list()->setEnabled(false);
 	MainWindow::instance()->setEnabledToolbar(false);
 	MainWindow::instance()->enterEditState();
+	ui.tabWidget->setTabEnabled(2, false);
+	ui.tabWidget->setTabEnabled(3, false);
+	ui.tabWidget->setTabEnabled(5, false);
```

Comment:
Simply pattern matching makes me wonder why we disable the fourth tab in one case and not the other - I guess I should actually read the code in context and try to figure it out myself...

Reply:
As the media tab makes sense on edit :-)

Reply:
it does? How?

Reply:
You can add media when editing a dive using the right mouse context menu ...

Reply:
I guess that shows you how often I use _THAT_ feature.